.github/workflows/main.yaml

```diff
@@ -72,13 +76,9 @@ jobs:
 
   publish-charm:
     name: Publish Charm
-    needs:
-      - lint-report
-      - static-analysis
-      - unit-tests-with-coverage
```

Comment:
Why we remove the those 3 jobs from needs before publishing charm ?

Reply:
I re-added them, reality is that what is really needed is integration tests. In any case I agree with re-adding them.